winter/routing/routing.py

```diff
@@ -14,8 +19,20 @@ def route(
         produces: Optional[Tuple[MediaType]] = None,
         consumes: Optional[Tuple[MediaType]] = None,
 ):
-    annotation = RouteAnnotation(url_path, http_method, produces, consumes)
-    return annotate(annotation)
+    route_annotation = RouteAnnotation(url_path, http_method, produces, consumes)
+    query_annotations = get_query_param_annotations(url_path)
+
+    def wrapper(func_or_method):
+        annotate_ = annotate(route_annotation, single=True)
+        func_or_method = annotate_(func_or_method)
+
+        for query_annotation in query_annotations:
+            annotate_ = annotate(query_annotation, unique=True)
```

Comment:
I think that it's not very obvious behavior to artificially add annotations to methods. What do you think? Can we use it in arguments resolvers and inspectors only?